### PR TITLE
DOCSUP-5917: Multiple names

### PR DESCRIPTION
**Pull request by olgarev**
Multiple names in CREATE, ALTER, SHOW CREATE for USERS, ROLES, ROW POLICIES, SETTINGS PROFILES, and SHOW CREATE QUOTAS

I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #11670 (part 1)

### CODE REVIEW

**Comment by olgarev** [comment]
@vitlibar Hi, will you review this, pls?
Unfortunately, I have not enough privileges for testing new syntax. 
May be, we should add some examples for multiple names? 
After checking and fixing mistakes I will edit the russian pages. 

Part 2 for the same PR is here https://github.com/ClickHouse/ClickHouse/pull/19488 (show access)
part 3 here https://github.com/ClickHouse/ClickHouse/pull/19486 (create quota, alter quota)

File: docs/en/sql-reference/statements/alter/role.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Changes roles.
 Syntax:
 
 ``` sql
-ALTER ROLE [IF EXISTS] name [ON CLUSTER cluster_name]
-    [RENAME TO new_name]
+ALTER {ROLE | ROLES} [IF EXISTS] name1 [ON CLUSTER cluster_name1] [RENAME TO new_name1]
```
Comment (vitlibar) [inline docs/en/sql-reference/statements/alter/role.md:13]: There is no command `ALTER ROLES`. Only `ALTER ROLE` exists, it just allows using multiple roles, kind of

```
ALTER ROLE role1, role2, ...
```

The same thing for other ALTER/CREATE commands.

**Comment by vitlibar** [comment]
> @vitlibar Hi, will you review this, pls?
done

> Unfortunately, I have not enough privileges for testing new syntax.
Can't you can't just install ClickHouse locally on your computer and test whatever you want?

File: docs/en/sql-reference/statements/alter/role.md (hunk 1 above)
Comment (olgarev) [reply docs/en/sql-reference/statements/alter/role.md]: @vitlibar Fixed that

**Comment by olgarev** [comment]
> > Unfortunately, I have not enough privileges for testing new syntax.
> 
> Can't you just install ClickHouse locally on your computer and test whatever you want?

Good idea )

**Pull request merged**